content/blog/2022-02-22-profiling.md

```diff
@@ -0,0 +1,208 @@
++++
+title = "Efficient Path Profiling"
+[[extra.authors]]
+name = "Niansong Zhang"
+link = "https://www.zzzdavid.tech"
+[[extra.authors]]
+name = "Jiajie Li"
+link = "https://tonyjie.github.io/"
++++
+
+<p align="center">
+<img src="https://res.cloudinary.com/dxzx2bxch/image/upload/v1646243822/cs6120/2_ugrqoi.png" alt="alt_text" title="image_tooltip" style="zoom:25%;" />
+</p>
+
+
+## INTRODUCTION
```

Comment:
I think the `<h1>` is plenty of emphasis: just use title case, like "Introduction" instead of "INTRODUCTION."